### ISSUE TITLE
Environment doesn't have y as variable

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/r-novice-gapminder#178]
> Managing your environment
> 
> There are a few useful commands you can use to interact with the R session.
> 
> `ls` will list all of the variables and functions stored in the global environment (your working R session):
> 
> ```
> ls()
> [1] "x" "y"
> ```

but we never defined `y` before.

**Comment by naupaka** [comment]
What lesson and line is this from?

**Comment by liz-is** [comment]
This occurs in [lesson1, line 389](https://github.com/swcarpentry/r-novice-gapminder/blob/gh-pages/_episodes_rmd/01-rstudio-intro.Rmd#L389).

**Comment by naupaka** [comment]
It's because of the script used to build all the lessons uses `x, y` as parameter values, so then these exist as objects inside the function for all of the lessons. See: https://github.com/swcarpentry/r-novice-gapminder/blame/gh-pages/bin/generate_md_episodes.R#L30

Example of the situation:

```r
test <- function(x, y) {
  # this is where all the knit() calls go
  print(ls())
}
test(1,2)
[1] "x" "y"
```

Not sure best way to get around this. Maybe easiest (although hacky) would be to just assign something to `y` so that the output of `ls()` isn't surprising.

**Comment by naupaka** [comment]
Closed by #282
